src/rsz/src/RepairSetup.cc

```diff
@@ -583,6 +617,26 @@ bool RepairSetup::swapPins(PathRef* drvr_path,
       return false;
     }
 
+    // Pass slews at input pins for more accurate delay/slew estimation
+    resizer_->input_slew_map_.clear();
+    std::unique_ptr<InstancePinIterator> inst_pin_iter{
+        network_->pinIterator(drvr)};
+    while (inst_pin_iter->hasNext()) {
+      Pin* pin = inst_pin_iter->next();
+      if (network_->direction(pin)->isInput()) {
+        LibertyPort* port = network_->libertyPort(pin);
+        if (port) {
+          Vertex* vertex = graph_->pinDrvrVertex(pin);
+          InputSlews slews;
+          slews[RiseFall::rise()->index()]
+              = sta_->vertexSlew(vertex, RiseFall::rise(), dcalc_ap);
+          slews[RiseFall::fall()->index()]
+              = sta_->vertexSlew(vertex, RiseFall::fall(), dcalc_ap);
+          resizer_->input_slew_map_.emplace(port, slews);
```

Comment:
It looks like this map will accumulate entries across all swap operations.  Would it be better to make this a local or clear it after each swap operation ends?

Reply:
The entries are cleared each time at line 621.

Reply:
Thanks I missed that